src/cache.js

```diff
@@ -2,6 +2,9 @@ import got from "got";
 import logger from "./logger.js";
 import { parseSchema } from "./parser.js";
 
+import { bootstrap } from "global-agent";
+bootstrap();
```

Comment:
Can you explain the thought process behind initialising this in cache.js?

I'd have thought loading it right at the top of the app bootstrap process (index.js) would make the most sense.

Reply:
It's the only place `got` (and therefore `global-agent`) is used. Wanted to keep them at the same place.

But as mentioned beforehand: my knowledge with node is limited. So feel free to move the code anywhere else.

Reply:
Sounds reasonable! Thanks for your feedback!
I just moved the code to `index.js` in 75d0696eb01a47d2828505b726ef269669b1aa3f